solutions/cf-1365D.mdx

```diff
@@ -269,14 +269,14 @@ for _ in range(int(input())):
 
 		cells_to_add = []
 		for new_i, new_j in [(i + 1, j), (i - 1, j), (i, j + 1), (i, j - 1)]:
-			if not (0 <= new_i < n and 0 <= new_j < m):
+			if (not (0 <= new_i < n and 0 <= new_j < m)
+					or visited[new_i][new_j]):
```

Comment:
uh what was this for

Reply:
check if the new cell is out of bounds so we don't throw an error. We can similarly skip if the cell is marked visited

Reply:
no like why did u change the formatting